frontend/src/main/components/Sections/SectionsInstructorTable.js

```diff
@@ -0,0 +1,92 @@
+import SectionsInstructorTableBase from "main/components/SectionsInstructorTableBase";
+
+import { yyyyqToQyy } from "main/utils/quarterUtilities.js";
+import {
+  convertToFraction,
+  formatDays,
+  formatInstructors,
+  formatLocation,
+  formatTime,
+  isSection,
+} from "main/utils/sectionUtils.js";
+
+export default function SectionsInstructorTable({ sections }) {
+  // Stryker enable all
+  // Stryker disable BooleanLiteral
```

Comment:
This is overly broad.   Please delete this and use something more surgical; Stryker exceptions should be rare, and this is likely to hide important testing coverage gaps.  If removing this causes an issue that you need help with, reach out about that specifically.
```suggestion
```

Reply:
Resolved! Removed the "disableGroupBy" var that was left over from SectionsTable.js too.